src/app/components/Curation/index.tsx

```diff
@@ -91,6 +95,8 @@ export default ({
       );
     case RADIO_SCHEDULE:
       return <RadioSchedule initialData={radioSchedule} />;
+    case EMBED:
+      return embed ? <Embed oembed={embed} type="" /> : null;
```

Comment:
👋 I was having a wee poke around in the OEmbed code, and I couldn't seem to find this `type` parameter being used. Perhaps a question for @amoore108, but would it be OK if we were to make `type` optional on [OembedProps](https://github.com/bbc/simorgh/blob/latest/src/app/components/Embeds/types.ts#L27-L30), so that we don't need to worry about hardcoding it to `""` here?

Reply:
Yea I had a play around with this too. I think `type` can be removed from `OEmbedProps`, its not used in the components that its typing.

This has a knock-on effect for the Flourish embeds tests: https://github.com/bbc/simorgh/blob/dbd29e743b43da179278c5c2b3733780fba093e7/src/app/components/Embeds/FlourishEmbed/index.test.tsx#L9-L16

☝️ They aren't actually typed correctly 😱 . They should be using `OEmbedData` instead, so if you change it to that it should help. You may need to use `} satisfies OEmbedData` to get it working.

I don't mind committing those changes if needed?

Reply:
@amoore108 @karinathomasbbc I've made the changes suggested - I had to update the width & height props on `OEmbedData` to support a string and number to keep the flourish tests happy as the iframe supports both of these types. Hope this is okay?

Reply:
Yeah that's perfect, thanks for fixing! 🙌🏻